src/main/java/net/pms/io/BasicSystemUtils.java

```diff
@@ -223,6 +223,11 @@ public String parsePingLine(String line) {
 		return timeString;
 	}
 
+	@Override
+	public int getPingPacketFragments(int packetSize) {
+		return ((packetSize + 8) / 1500) + 1;
+	}
```

Comment:
Do the arbitrary MTU choice was done because it's cumbersome to acces the different OS registries, or because it'll not make much difference in the speed estimation anyway ?